robottelo/cli/base.py

```diff
@@ -86,6 +85,33 @@ class Base(object):
     logger = logging.getLogger('robottelo')
 
     @classmethod
+    def _handle_response(cls, response, ignore_stderr=None):
+        """Verifies ``return_code`` and ``stderr`` in ``SSHCommandResult``
+        response and throws ``CLICommandExecutionError`` exception if
+        ``return_code`` is not equal to 0.
+
+        """
+        if response.return_code != 0:
+            raise CLICommandExecutionError(
+                response.return_code,
+                response.stderr,
+                'Command \'{0} {1}\' finished with return_code {2}\n'
```

Comment:
You can use single and double quotes together to get rid of these back-slashes.

    "Command '{0} {1}' finished with..."